### PR TITLE
feat: example upload file to swarm in react

### PR DESCRIPTION
**Pull request by vojtechsimetka**

### CODE REVIEW

File: upload-react/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```
Comment (AuHau) [inline upload-react/README.md:1]: Maybe we could change the README a bit from this boilerplate? For example describe what are you doing in the example, what Bee(JS) APIs are you using, and how? Example of my README for the MetamaskSigning: https://github.com/ethersphere/bee-js/pull/168/files#diff-9576d5f38c5e309082bc2573cef0e58eb5bfd6e1a44239aba036ee64686b12e2

Comment (vojtechsimetka) [reply upload-react/README.md]: Sorry I haven't seen this. This sounds like a good idea and I'll add it but will not be able to do so until 3PM.

File: upload-react/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>)=> {
+    event.preventDefault();
+
+    if (file) {
+      try {
+      setUploading(true)
+      setLink(null)
+
+      const hash = await bee.uploadFile(file)
+      setLink(`${beeUrl}/files/${hash}`)
```
Comment (agazso) [inline upload-react/src/App.tsx:23]: Missing indentation

File: README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+> Examples for [bee-js](https://github.com/ethersphere/bee-js) and other Swarm JS projects
+
+**Warning: These exaples are using libraries and tools under heavy development. There might (and most probably will) be changes in the future to its API and working. Also, no guarantees can be made about stability, efficiency, and security at this stage.**
+
+## Examples
+
+| status | name                           | description                      |
+|---     |---                             |---                               |
+| OK     | [pss-react](./pss-react)       | Send and receive pss messages    |
```
Comment (agazso) [inline README.md:11]: Currently there is no `pss-react` example

File: README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+| OK     | [upload-react](./upload-react) | Simple file upload dApp in React |
+
+
+## License
+
+[BSD-3-Clause](./LICENSE)
```
Comment (agazso) [inline README.md:17]: Maintainers are missing

**Review by agazso — APPROVED** [review]
Looks good, but see my comments

File: upload-react/README.md (hunk 1 above)
Comment (AuHau) [reply upload-react/README.md]: Sounds good

**Comment by AuHau** [comment]
I have implemented the comments, so I will merge it.

**Pull request merged**